src/features/events/components/newsletter/newsletter.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import NewsletterForm from './'
+
+describe('<NewsletterForm/>', () => {
+  beforeAll(() => {
+    global.___loader = {
+      enqueue: jest.fn(),
+    }
+    global.fetch = jest.fn().mockImplementation(() => Promise.resolve({}))
```

Comment:
To avoid modifying the `global` scope, a library such as [`jest-fetch-mock`](https://www.npmjs.com/package/jest-fetch-mock) would work nicely.

Reply:
Deleted tests per our conversation about this component being deprecated for time being